internal/integration/testdata/mysql/time-type-precision-maria.txt

```diff
@@ -0,0 +1,45 @@
+# Each test runs on a clean database.
+only maria
+
+# Apply schema "1.hcl" on fresh database.
+apply 1.hcl
+
+# Compare the result of "SHOW TABLE users" with the content of a file named '1.sql'.
+# The "cmpshow" command searches first a file named '<version>/1.sql' (version, defines
+# the database version), and in case it was found, it will use it instead.
```

Comment:
I don't think this comment is needed here. I created one documented example test, but the rest can be without it.